examples/stack/sample-aws-stack-nb.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
micronit: let's replace the other example with just this one.

Reply:
sounds good.